homeassistant/components/motioneye/config_flow.py

```diff
@@ -225,7 +225,7 @@ async def async_step_init(
             # the template input won't be a valid URL until after it's rendered.
             schema.update(
                 {
-                    vol.Required(
+                    vol.Optional(
                         CONF_STREAM_URL_TEMPLATE,
                         default=self._config_entry.options.get(
                             CONF_STREAM_URL_TEMPLATE,
```

Comment:
Line below should be `None` instead of an empty string

Reply:
You need to change the line below the one I have tagged to `None`. I cannot tag the lines as Github only allows me to tag 3 lines from the thing you have changed 😃

Reply:
> You need to change the line below the one I have tagged to `None`. Right now it is an empty string `""`. I cannot tag the lines as Github only allows me to tag 3 lines from the thing you have changed 😃

Sorry, I’m a bit confused - I’ve only touched one line in the actual config flow. I can see line 232 had the `""` before my PR - Does that need to change to `None` due to the change from `Required` to `Optional`?

Reply:
Yes and because you also have changed the tests.